includes/api/prepay-cod.php

```diff
@@ -1,6 +1,133 @@
 <?php
 
+const WC_ORDER_ID                    = 'woocommerce_order_id';
+const RAZORPAY_PAYMENT_ID            = 'razorpay_payment_id';
+const RAZORPAY_ORDER_ID              = 'razorpay_order_id';
+const RAZORPAY_SIGNATURE             = 'razorpay_signature';
+const RAZORPAY_OFFER                 = 'razorpay_offer';
+
 function prepayCODOrderHandler(WP_REST_Request $request) {
     $params = $request->get_params();
     return prepayCODOrder($params);
 }
+
+
+function prepayCODOrder(array $payload): WP_REST_Response {
```

Comment:
We can directly set this function as callback right? why do we need `prepayCODOrderHandler` function ?